examples/event-webhooks-local-flask/README.md

```diff
@@ -0,0 +1,278 @@
+
+It relies on a few other tools that will be installed for you within Docker containers:
+* [Localtunnel](https://localtunnel.me/) - expose a public webhook URL and forward the results locally. ⚠️ *Not for production or real data!*
+* [Flask](https://flask.palletsprojects.com/) - A simple Python web application framework
+* [PostgreSQL](https://www.postgresql.org/) - A free, open source database representing an external data store example
+
+## Getting Started
+
+Create an empty placeholder file for Docker secrets. *nix example:
+
+```bash
+touch .client_secret
+touch .database_password
+```
+
+Windows example:
+
+```cmd
+echo.> .client_secret
+echo.> .database_password
+```
+
+A database password can be created now. It can be any valid PostgreSQL password. One way to generate a random password for local development purposes on *nix systems might be:
+
+```bash
+date +%s | sha256sum | base64 | head -c 16 | cat > .database_password
+```
+
+You can also open the file and manually enter any password for local development. In production, be sure to generate and store passwords securely.
+
+> ⚠️ **Security Note:** Be sure to avoid committing `.database_password` to a source code repository.
+
+Start Docker:
+
+```bash
+docker compose up --build -d
+```
+
+Tip: You can omit the `-d` option if you want to run in the foreground. Otherwise, use `docker compose logs -f` to tail logs.
+
+> ℹ️ **Windows Note 1:** "Use ContainerD for pulling and storing images" may need to be enabled in `Docker > Settings > Features in development > Beta Features`
+
+> ℹ️ **Windows Note 2**: If running into an error like "ERROR: request returned Bad Gateway for API route and version", [this solution](https://github.com/docker/for-mac/issues/6956#issuecomment-1876444658) may fix the problem.
+
+
+You can verify that Flask is up and running:
+
+```bash
+curl localhost:8000/health
+```
+
+If Flask is running, you should see `OK` printed.
+
+Be sure to note the URL created for you by `localtunnel`. The log line should look something like this:
+
+```
+local-tunnel-1   | your url is: https://brave-wombats-poke.loca.lt
+```
+
+On *nix systems, you can easily obtain _just_ the URL via:
+
+```
+docker compose logs local-tunnel | grep -o https://.* | tail -n 1
+```
+
+Example Output:
+
+```
+https://brave-wombats-poke.loca.lt
+```
+
+> 💡 Don't forget to append `/1/webhooks`, making the full URL given to Benchling `https://brave-wombats-poke.loca.lt/1/webhooks`
+
+## Setting Up Your App in Benchling
+
+### Benchling Prerequisites
+1. Access to a Benchling tenant, like `https://my-tenant.benchling.com`
+2. Ensure you've been granted access to the [Benchling Developer Platform Capability](https://help.benchling.com/hc/en-us/articles/9714802977805-Access-the-Benchling-Developer-Platform).
+3. This example also requires Events delivered by Webhooks to be enabled on your tenant. Reach out to Benchling support to find out more about participating in early access.
```

Comment:
@Psterne @dgervais-benchling similar to above, let me know if you want different copy to express this requirement.

Reply:
Looks good. Is there a step 4 that's missing? Should Global Apps be step 4?

Reply:
The number skipped 4 and went to 5. Fixed now, thanks for catching!